### PR TITLE
Bumped python version from 3.8->3.8.13

### LINKED ISSUE #38: Security vulnerabilities found on py-runner 2.7.0
## Details:

- We found 13 critical vulnerabilities on py-runner and need to bump python to the latest version: 3.8.13 to resolve those vulnerabilities.

### PR DESCRIPTION
**Pull request by rociomroman**
Signed-off-by: rociomroman [rociosiuce@gmail.com](mailto:rociosiuce@gmail.com)

**What this PR does / why we need it**:

- Fix the 13 vulnerabilities found on py-runner

**Which issue this PR fixes** : fixes #38 

**Special notes for your reviewer**: N/A

**Checklist:**
-   [x] Fixes #38 
-   [ ] PR messages has document related information
-   [ ] Labelled this PR & related issue with `breaking-changes` tag
-   [ ] PR messages has breaking changes related information
-   [ ] Labelled this PR & related issue with `requires-upgrade` tag
-   [ ] PR messages has upgrade related information
-   [ ] Commit has unit tests
-   [ ] Commit has integration tests
-   [ ] E2E run Required for the changes

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.8
+FROM python:3.8.13
```
Comment (DerPdeFeNdeR) [inline Dockerfile:1]: +100000 :)

**Comment by neelanjan00** [comment]
Hi @rociomroman, thank you so much for the contribution. The DCO check is failing, can you please sign your commit with the provided instructions (in Details)?

File: Dockerfile (hunk 1 above)
Comment (erekgit) [reply Dockerfile:1]: Seconding @DerPdeFeNdeR's +100000 :)

**Comment by uditgaurav** [comment]
Thanks for the PR @rociomroman !

**Review by uditgaurav — APPROVED** [review]

**Comment by rociomroman** [comment]
@ksatchit and @ispeakc0de, can I please get some additional eyes on this PR? thank you!

**Comment by ksatchit** [comment]
@uditgaurav can we please fix the synk task failure here!

**Comment by ksatchit** [comment]
Looks good @rociomroman ! Just checking if you managed to run a sample experiment with the modified (custom) image? We still do not have integration/e2e tests set up for this repo!

**Comment by rociomroman** [comment]
Hi @ksatchit  thank you for reviewing my PR.  Is there a possibility that we can validate this python upgrade on some external environment? I could use some assistance on the validation part so any help is much appreciated

**Comment by oumkale** [comment]
Hi @rociomroman,

Thank you for the PR,

3 ways to test the upgrade scenario,
1. Using okteto: https://github.com/litmuschaos/litmus-python/tree/master/contribute/developer-guide#steps-to-test-experiment (ignore unwanted steps like updating codes)
  - Run: curl -L https://storage.googleapis.com/kubernetes-release/release/"v1.18.0"/bin/linux/"amd64"/kubectl -o /usr/local/bin/kubectl && chmod +x /usr/local/bin/kubectl
 - Run: pip3 install -r requirements.txt
 - Run: python3 setup.py install, run this command for every change in experiment code.
 - python3 bin/experiment/experiment.py
 
 2. Using local env:
  https://dev.to/oumkale/create-chaos-experiments-using-the-litmuschaos-python-sdk-4492
   - Ignore for experiment writing part, use only setup steps:
   - Let’s come bank to root directory litmuschaos/litmus-python to setup environment.
   - python3 -m virtualenv chaos
   - source chaos/bin/activate
   - pip3 install -r requirements.txt
   - python3 setup.py install 
   - kubectl create deployment nginx --image=nginx
   - python3 experiment.py 
  
  3. You may build docker image: 
  Experiment: Update your image in `image` https://github.com/litmuschaos/chaos-charts/blob/master/charts/generic/pod-delete/python/experiment.yaml
 
And run experiment as usual: 
 1. Run it locally by installing operator, rbac, nginx, chaosexperiment and chaosengine
 Look into this: https://github.com/litmuschaos/chaos-charts/tree/master/charts/generic/pod-delete/python
 2. Run From chaos centre: select pod-delete experiment update experiment part and run it or upload workflow manifest which is updated with new experiment.

Let us know if you face any issue!!

**Comment by uditgaurav** [comment]
@rociomroman can you please update the branch!

**Review by oumkale — APPROVED** [review]

**Comment by ksatchit** [comment]
PR #41 is merged

**Comment by rociomroman** [comment]
@ksatchit Awesome. Now that PR #41 is merged, is there anything else on my end that I have to do in order to get this PR merged? thanks

**Review by ksatchit — APPROVED** [review]

**Comment by oumkale** [comment]
Hi @rociomroman,
Thank you for the PR, Good to take this PR.
You may reset commits

**Pull request merged**